### ISSUE TITLE
Cropping the last (milli)seconds

### ISSUE THREAD

**Issue by fb87** [phw/peek#31]
I thought, maybe it's useful to provide an option to crop the last hundreds of milliseconds (or seconds) at the end of the recording, because they often just contain moving the mouse to the „stop“ button. Or when the user sees, that nothing changed in the last seconds and therefore stops the recording, he/she can crop this superfluous part. This saves space and upload time, especially at higher resolutions like UHD or when big parts of the screen are captured.

**Comment by phw** [comment]
> ...the last hundreds of milliseconds (or seconds) at the end of the recording, because they often just contain moving the mouse to the „stop“ button.

That's definitely something annoying me seriously :) My preferred solution to this actually is to support keyboard shortcuts (see #23 ), but the crop option could be added additionally.

**Comment by fb87** [comment]
It just came to my mind, that a timeline view could be shown after clicking on e.g. a „edit/crop/etc“ button, when the screencast is finished. In this timeline, the start and end could be moved via a slider like in video editing software, and even pauses (#40) should be made possible. :)

And maybe start/stop could be controllable by a hotkey, so that you don't need mouse action, if this is applicable in the specific situation.

**Comment by srichakradhar** [comment]
@phw I guess now we have a keyboard shortcut for this.. `Ctrl + Alt + R`

**Comment by phw** [comment]
@srichakradhar Yes, and that keyboard shortcut can be configured in the preferences.

Global keyboard shortcuts don't work for Wayland, but if the desktop environment / compositor allows configuring global shortcuts you can get the same functionality by binding a shortcut to `peek --toggle`. E.g. I have configured this on GNOME Wayland:

![bildschirmfoto vom 2017-11-23 10-21-22](https://user-images.githubusercontent.com/29852/33165710-180a6982-d038-11e7-9e9d-6bf0f00d73b7.png)


I should document this somewhere :)

**Comment by phw** [comment]
I am not sure about the crop function, but having #14 would probably solve this. I am inclined to close this issue, but will think about #14 first a bit.

**Comment by fbruetting** [comment]
Jeah, could be just a shortcut like “cut off the last second”, because that would be of use very often – so the people don’t need to search for the correct frame from which to crop.
